addons/mail/controllers/discuss/avatar_card.py

```diff
@@ -0,0 +1,25 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.mail.tools.discuss import add_guest_to_context
+
+
+class AvatarCardController(http.Controller):
+    @http.route("/discuss/avatar_card", methods=["POST"], type="jsonrpc", auth="public")
+    @add_guest_to_context
+    def avatar_card(self, user_id, partner_id, fields):
+        if not user_id and partner_id:
+            partner = request.env["res.partner"].browse(partner_id).exists()
+            if not partner:
+                return False
+            users = partner.user_ids.filtered(lambda u: u.active).with_prefetch(partner.user_ids.ids)
```

Comment:
I believe active_test is true by default, there is no need to explicitly filter active?
Also what's the point to explicitly re-set the prefetch to included archived one?